nx.json

```
@@ -47,5 +39,10 @@
       "style": "scss"
     }
   },
-  "defaultProject": "lumberjack-app"
+  "defaultProject": "lumberjack-app",
+  "targetDefaults": {
+    "build": {
+      "dependsOn": ["^build"]
```

suggestion: Use the explicit format to make it clearer.
```suggestion
      "dependsOn": [{ "projects": "dependencies", target: "build" }]
```